HackIllinois.xcodeproj/project.pbxproj

```diff
@@ -1332,7 +1332,7 @@
 					"$(inherited)",
 					"@executable_path/Frameworks",
 				);
-				MARKETING_VERSION = 2022.1.2;
+				MARKETING_VERSION = 2024.1.2;
```

Comment:
The current version in the App Store is 2024.1.2 so when you deploy these changes it'll be 2024.1.3